darwin/cli_functions.py

```diff
@@ -546,7 +546,8 @@ def dataset_convert(dataset_slug, format, output_dir):
         annotations_path = release_path / "annotations"
         if output_dir is None:
             output_dir = release_path / "other_formats" / f"{format}"
-            output_dir.mkdir(parents=True, exist_ok=True)
+        output_dir = Path(output_dir)
```

Comment:
`output_dir` should already be a `Path`, or at least that's what we seem to do in line 548. 

If that's not the case, it's because we call `dataset_convert` with an `output_dir` of a different type (likely a string)

I would suggest:
1. annotate the function definition with `typing`
2. make sure we always call this function with the right types (`mypy` would help at finding mistakes on our end)

Reply:
Yes, you're right. I annotated the function and removed that line.